springerlink.md

```diff
@@ -1,163 +1,1552 @@
+<li>11.2.1.33 (Software)</li>
+<li>11.2.2.33 (Closed Software)</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Where user input errors are automatically detected in our own software, the items that are in error are identified to the user with warning text adjacent to the input field. We do not rely on visual characteristics alone (e.g. colour) to indicate user input errors. 
+<br><br>
+In some parts of the site, we additionally use an aria-describedby property to associate error messages with invalid fields.
+<br><br>
+User account pages located at link.springer.com/account and pages located at checkout.springer.com unsemantically identify errors in label elements, which may be problematic for some types of Assistive Technology. 
+</td>
+</tr>
+<tr id="labels-or-instructions" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#minimize-error-cues"><strong>3.3.2 Labels or Instructions</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.34 (Web)</li>
+<li>10.2.34 (non-web document)</li>
+<li>11.2.1.34 (Software)</li>
+<li>11.2.2.34 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+The SpringerLink website contains content that is rendered in two separate designs. We are in the process of transferring all content from the older design to the newer design. 
+<br><br>
+Pages rendered in the <strong>older</strong> design (currently Journals, Book Series, and ancillary pages such as search, indexes, and account management) generally display visible and appropriately associated labels for input fields. Exceptions include the pagination number field on advanced search results pages and in the main search component in the page banner, both of which omit any label or ARIA description, and use only placeholder text as instruction. We are actively working to find and repair these problems.  
+<br><br>
+Pages rendered in the <strong>newer</strong> design (currently Articles, Chapters, Books, Reference Works, and Reference Work Entries) generally display visible and appropriately associated labels for input fields. 
+<br><br>
+Instructions such as those that explain mandatory fields may be absent or only present at the end of forms. (e.g. some payment information pages). Additional instructions for specific input fields may not be adequately associated (e.g. with aria-describedby) with the fields they describe. We are actively working to find and repair these problems. 
```

Comment:
Could format aria-describedby as `aria-describedby`? It is also mentioned in one other place.

Reply:
did a sweep and found some other attributes that could have been formatted too 😬